tensorflow/lite/kernels/internal/reference/reference_ops.h

```diff
@@ -2635,49 +2635,6 @@ T FloorMod(T input1, T input2) {
              : trunc_mod;
 }
 
-inline void Floor(const RuntimeShape& input_shape, const float* input_data,
```

Comment:
For internal reasons, we need to keep the Floor/Ceil/Round implementations in this file. Can you please update accordingly? Thanks.